fix: make it explicit that gateway-api only works in standalone (#1021)
kumahq/kuma-website: docs/docs/dev/explore/gateway-api.md

::: warning
[Kubernetes Gateway API](https://gateway-api.sigs.k8s.io/) is still beta, therefore Kuma's integration provides the same level of stability.

> gateway-api is not supported in multi-zone. To do this you need to use [Kuma MeshGateway](../gateway.md)

Reviewer: ```suggestion
Gateway API is not supported in multi-zone. To use the builtin Gateway, you need to use the [`MeshGateway`  resources](../gateway.md).
```
